ts/store/reducers/index.ts

```diff
@@ -80,7 +81,8 @@ const appReducer: Reducer<GlobalState, Action> = combineReducers<
   notifications: notificationsReducer,
   profile: profileReducer,
   entities: entitiesReducer,
-  debug: debugReducer
+  debug: debugReducer,
+  pagoPAEnv: pagoPAEnvReducer
```

Comment:
I believe we want the setting to be persisted, I suggest to move the setting under `persistedPreferences` and avoid creating a new reducer